custom_components/better_thermostat/models/models.py

```diff
@@ -160,16 +160,16 @@ def convert_outbound_states(self, hvac_mode) -> Union[dict, None]:
 					_LOGGER.warning(
 						f"better_thermostat {self.name}: device config expects no system mode, while the device has one. Device system mode will be ignored"
 					)
-				hvac_mode = None
 				if hvac_mode == HVAC_MODE_OFF:
+					hvac_mode = None
 					_new_heating_setpoint = 5
 			
 			elif _has_system_mode is None and _system_mode is None:
-				hvac_mode = None
```

Comment:
If a TRV has an empty system mode and we don't have no info about system modes, it's safe to assume, that we don't want to send a system mode. So why is this a bug?

Reply:
It's in the wrong line? isn't it?

If we check "if hvac_mode == HVAC_MODE_OFF" after we set hvac_mode to none its never fired.

Reply:
None isn't HVAC_MODE_OFF :)

Reply:
None is the signal for us, that we won't initiate an of the system mode in the "upper layer" which parses the result, but instead just send the setpoint.

Reply:
https://github.com/KartoffelToby/better_thermostat/blob/7f443877344d08482a6ddfa5665d1426603b2cdc/custom_components/better_thermostat/controlling.py#L36

Reply:
But I see the issue, I pushed a fix. :)